libs/media/src/lib/video/viewer/viewer.component.ts

```diff
@@ -163,10 +164,14 @@ export class VideoViewerComponent implements AfterViewInit, OnDestroy {
         this.player.on('complete', () => this.stateChange.emit('complete'));
         this.updatePlayer();
       }
-    } catch (error) {
+    } catch (err) {
       this.loading$.next(false);
-      console.warn(error);
-      this.snackBar.open('Error while playing the video: ' + error, 'close', { duration: 8000 });
+      console.warn(err);
+      if (err.message === 'There was a problem loading this video...') {
+        this.snackBar.openFromComponent(SnackbarErrorComponent, { data: err.message, duration: 7000 });
+      } else {
+        this.snackBar.open(err.message, 'close', { duration: 8000 });
+      }
```

Comment:
"There was a problem loading this video.." is not supposed to be displayed no matter the error ?
```suggestion
       this.snackBar.openFromComponent(SnackbarErrorComponent, { data: 'There was a problem loading this video...', duration: 7000 });

```

Reply:
I did this to make it clear to the user if the problem was with the watermark, I modified it to make a global message